salt/metalk8s/kubernetes/coredns/deployed.sls

```diff
@@ -50,8 +50,6 @@ Create coredns service:
           prometheus.io/scrape: "true"
         labels:
           k8s-app: kube-dns
-          kubernetes.io/cluster-service: "true"
-          kubernetes.io/name: "CoreDNS"
```

Comment:
Replace this one with `app.kubernetes.io/name: coredns`

Reply:
`app.kubernetes.io/name: "CoreDNS"`